Dashboard/src/config.js

```diff
@@ -7,5 +7,6 @@ export default {
   presentations: `http://${dashHost}:8083`,
   meetings: `http://${dashHost}:8084`,
   infrastructure: `http://${dashHost}:8085`,
-  github: `http://${dashHost}:8086`
+  github: `http://${dashHost}:8086`,
+  reminder: `http://${dashHost}:8086/pull-requests`
```

Comment:
The dashboard no longer uses this config file (we should probably remove it). It will instead automagically fetch from the proper container in the Docker network based on the panel's name (i.e. `http://data/{container-name}`). I believe the `getData` function in `Service.js` will need to be changed to support fetching from a specific `path` (e.g. `pull-requests`), since we would previously only fetch from the `root` endpoint of each container.

Reply:
Let's make removing `Dashboard/src/config.js` part of a separate issue.